src/Common/LoggingFormatStringHelpers.h

```diff
@@ -191,6 +191,34 @@ class LogFrequencyLimiterIml
     Poco::Logger * getLogger() { return logger; }
 };
 
+/// This wrapper helps to avoid too noisy log messages from similar objects.
+/// For the value logger_name it remembers when such a message was logged the last time.
+class LogSeriesLimiter
```

Comment:
Maybe it worth do underline the difference between `LogFrequencyLimiterIml` in the comment? (and the helpers macros)

Reply:
I'm on my way to do it.